File: backend-model/src/main/java/outcobra/server/model/dto/HolidayDto.java
package outcobra.server.model.dto;

import outcobra.server.model.Holiday;
import outcobra.server.model.interfaces.MappableDto;
import outcobra.server.model.interfaces.Mapper;

public class HolidayDto implements MappableDto<HolidayDto, Holiday> {
    @Override
    public Mapper<HolidayDto, Holiday> getMapper() {
        throw new UnsupportedOperationException("not implemented");
    }

    @Override
    public Holiday toEntity() {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: backend/src/main/kotlin/outcobra/server/model/dto/HolidayDto.kt
package outcobra.server.model.dto

import outcobra.server.model.mapper.MappableDto
import outcobra.server.model.Holiday
import outcobra.server.model.mapper.Mapper

/**
 * Created by Florian on 04.11.2016.
 */
class HolidayDto : MappableDto<HolidayDto,Holiday> {
    override fun getMapper(): Mapper<HolidayDto, Holiday> {
        throw UnsupportedOperationException("not implemented")
    }

    override fun toEntity(): Holiday {
        throw UnsupportedOperationException("not implemented")
    }
}